mopidy_mpd/dispatcher.py

```diff
@@ -248,6 +251,8 @@ def __init__(
         self.session = session
         if config is not None:
             self.password = config["mpd"]["password"]
+            # TODO: don't default here.
+            self.multiple_tags = config["mpd"].get("multiple_tags", False)
```

Comment:
The `multiple_tags` key does not always exist in `config["mpd"]` in the tests. It is now optional but set in the default configuration. Should the tests be changed, a default value be set here (as it is right now), or something else?

Reply:
I think it's appropriate to update the tests.